app/src/ui/branches/branches-container.tsx

```diff
@@ -115,6 +115,7 @@ export class BranchesContainer extends React.Component<
             defaultBranch={this.props.defaultBranch}
             currentBranch={this.props.currentBranch}
             allBranches={this.props.allBranches}
+            isBranchListDropdown={true}
```

Comment:
You can actually move the button write after this component. Doing that will eliminate the need to pass the `isBranchListDropDown` prop to the `BranchList` component. 

https://github.com/desktop/desktop/blob/9aaf2a1d0ae39ea0500dfa4b5922db2d22a67b7e/app/src/ui/branches/branches-container.tsx#L112-L128